docs/faqs/NuGet-FAQ.md

```diff
@@ -0,0 +1,387 @@
+
+## nuget.org not accessible
+
+**Why can't I download packages from or upload packages to nuget.org?**
+
+First, make sure you're using the latest versions of NuGet. If that version continues to fail, [contact support](https://www.nuget.org/policies/Contact) and provide additional connection troubleshooting information including:
+
+- The version of NuGet you're using
+- The package sources you're using
+- A restore log with detailed verbosity
+- MTR or a Fiddler traces (see below)
+- Your geographical area
+- Your operating system version
+- Machine configuration (CPU, Network, hard drive)
+- Whether your machine is behind a proxy or firewall
+- The versions of .NET that are installed on the machine
+- Versions of cross-platform tools such as .NET CLI, or DNU that you're using
+
+*To capture MTR:*
+
+- Download WinMTR from [http://winmtr.net/download/](http://winmtr.net/)
+- Enter `api.nuget.org` as the hostname and click **Start**.
+- Wait until the **Sent** column is >= 100.
+
+    ![Capturing MTR](media/mtr.png)
+
+- Copy text to clipboard.
+
+*To capture Fiddler:*
+
+- Install the latest version of [Fiddler](http://www.telerik.com/download/fiddler).
+- Start Fiddler and disable capturing traffic using the **File > Capture Traffic** menu.
+- Remove all sessions (select all items in the list, press the **Delete** key).
+- Configure Fiddler to capture HTTPS traffic by checking **Decrypt HTTPS traffic** in the **HTTPS** tab of the **Tools > Fiddler Options...** menu.
+- Close Visual Studio.
+- Enable the **File > Capture Traffic** menu.
+- Start Visual Studio or nuget.exe .exe and perform the actions that are not working. The traffic generated by these actions should show up in Fiddler.
+- Once the actions have run, use **File > Save > All Sessions** to store the captured sessions.
+
+Note: it may be required to set the `HTTP_PROXY` environment variable to `http://127.0.0.1:8888` for routing NuGet traffic through Fiddler.
+
+If that fails, try the [tips mentioned in this StackOverflow post](http://stackoverflow.com/questions/21049908/using-fiddler-to-sniff-visual-studio-2013-requests-proxy-firewall).
+
+**What are the API endpoints for nuget.org?**
+
+- V3: `https://api.nuget.org/v3/index.json`
+- V2: `https://www.nuget.org/api/v2/` (Note that the V2 API is deprecated and does not work with NuGet 4+.)
+
+## NuGet.org Account Management
+
+### How to create a new NuGet account?
+
+To create a NuGet account, you need to have a personal Microsoft account (MSA) or an Azure Active Directory (AAD) account. If you do not have one, you can [create](https://www.microsoft.com/en-us/account) one. Follow the following steps if you have a MSA or AAD account.
+- Go to the [login page](https://www.nuget.org/users/account/LogOn?returnUrl=%2F).
+- Click on `Sign in with Microsoft` button.
```

Comment:
Using backticks (code) for button labels feels a little off. Could we just bold?